Module3_2/Module3_2/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+
+namespace Module3_2
+{
+    class Program
+    {
+        static void Main()
+        {
+            int naturalNumberFromUser = GetNumber();
+            int amountOfNaturalEvenNumbers = GetAmountOfNaturalEvenNumbers();
+
+            calculateAListOfNaturalNumbers(naturalNumberFromUser, amountOfNaturalEvenNumbers);
+            Console.ReadKey();
+        }
+
+        static int GetNaturalNumberFromUser(string messageToUser)
+        {
+            bool numberIsValid;
+            bool numberIsPositive;
+            int naturalNumber;
+
+            do
+            {
+                Console.WriteLine(messageToUser);
+                string response = Console.ReadLine();
+                numberIsValid = int.TryParse(response, out naturalNumber);
```

Comment:
boolean variable should starts with such words as is, has, can etc. In your case isNumberValid